ember-app/app/components/hb-column-crumb.js

```diff
@@ -5,11 +5,11 @@ var HbColumnCrumbComponent = Ember.Component.extend({
   classNames: ['crumb'],
   classNameBindings: ["stateClass", "isSelected:active:inactive", "indexClass"],
   isSelected: function(){
-    if(this.get("issue.data.state") === "closed"){
-      return this.get("visibleColumns.lastObject.data.name") === this.get("column.data.name");
-    }
-    return this.get("issue.data.current_state.name") === this.get("column.data.name");
-  }.property("issue.data.current_state.name", "column.data.name", "issue.data.state"),
+    var current_issue = this.get("issue");
+    return this.get("column.sortedIssues").find((issue)=> {
+      return issue.get("id") === current_issue.get("id");
```

Comment:
Is there any measurable efficiency to be gained from capturing `current_issue.get("id")`?

```javascript
var current_id = this.get("issue").get("id");
return this.get("column.sortedIssues").find((issue)=> {
 return issue.get("id") === current_id;
});
```

And another Ember newbie question: would `this.get("issue.id")` work?